src/autopas/LogicHandler.h

```diff
@@ -153,11 +152,9 @@ class LogicHandler {
    * @copydoc AutoPas::updateContainer()
    */
   [[nodiscard]] std::vector<Particle> updateContainer() {
+    this->checkNeighborListsInvalidDoDynamicRebuild();
```

Comment:
```suggestion
    // update member variable _neighborListInvalidDoDynamicRebuild
    this->checkNeighborListsInvalidDoDynamicRebuild();
```

Reply:
also place a timer around this function and add it to your measurements.

Reply:
time for checking rebuild condition is reflected in increased time for Update Container call. I have not added a separate timer for it in AutoPas as all our timers are in `iteratePairwise()` function and this condition is checked in `updateContainer()`. I would need to find a way to return it and add it to logger. But as this can already be inferred, I have skipped it for now. I could look at implementing this if deemed absolutely necessary.